tests/conftest.py

```diff
@@ -98,6 +98,12 @@ def app_config(app_config):
     app_config["RECORDS_REST_ENDPOINTS"] = {}
     app_config["INDEXER_DEFAULT_DOC_TYPE"] = "testrecord"
     app_config["INDEXER_DEFAULT_INDEX"] = TestSearch.Meta.index
+    app_config["RECORDS_UI_ENDPOINTS"] = {
```

Comment:
Maybe? :) Taking the UI route from there does indeed close: #55 . 

It would be nicer if routes were defined together, but I don't think the Resource should know the UI route right now... 

I had a shower-thought about actually using the Service as the source of truth for routes. It would require some little changes, but that's a possibility to simplify where routes should be and prevent potential duplication...